tests/command.help.test.js

```diff
@@ -130,3 +130,20 @@ test('when both help flags masked then not displayed in helpInformation', () =>
   const helpInformation = program.helpInformation();
   expect(helpInformation).not.toMatch('display help');
 });
+
+test('when default value is object with toString and displays in helpInformation', () => {
```

Comment:
1) The test naming is _when_ setup _then_ what is tested. 
2) suggest "with custom toString"

e.g. 'when default value is object with custom toString then custom string displays in helpInformation'